docs/technical-design/contract-rate-change-history.md

```diff
@@ -2,29 +2,49 @@
 title: Calculating contract and Rate Change History
 ---
 
-## Calculating contract and Rate Change History
+## Calculating Contract and Rate Change History
 
 ## Overview
-Change history is a feature of MC-Review where we track changes to the submission data over time and display this content to users. This document details how the change history is calculated for contract and rate data and which database fields are used.
+Change history is the feature of MC-Review where the application stores full copies of changes to submission data over time. The data is displayed to users on the submission summary page.
 
-[ADD IMAGE]
+ ![Change History on Submission Summary page](../../.images/change-history-feature-ui.png)
+
+ This document details how the change history is calculated for contract and rate data and which database fields are used.
 ## Contraints
-- MC-Review must track actions on contract or a rate (submit, unlock, resubmit) alongside the full form data present at that that momment in the database. This data is used for CMS reporting and audit purposes. It is considered part of the system of system.
-- From a product requirement standpoint, MC-Review does not need to track changes on drafts (each edit a state makes to a draft data directly updates the original resource).
+- MC-Review must track version history once a contract or rate is submitted.
+    - This includes all actions on contract or a rate (submit, unlock, resubmit) alongside the related form data present at that point in time in the database.
+    - This data is used for CMS reporting and audit purposes. It is considered part of the system of record.
 
 ## Implementation
-### A full copy of the submission data at a given point in time is called a revision
-The change history audit log is a list of revisions sorted by data. This is currently stored in the `revisions` field on the Contract and Rate tables.
+### The full copy of the contract or rate  data at a given point in time is called a *revision*. A new revision is added on create or unlock. These are stored in a `revisions` field on the Contract or Rate.
+
+The `revisions` field on the Contract and Rate tables is the most important part of our version history. The user can only ever change one revision at a time. Sorting of revisions is by `createdAt` - so the point in time when the revision was created by a state submit or a CMS unlock.
+### Drafts are not part of the revisions list
+
+Change history does not include information about drafts. Accordingly, draft data being worked on by the state is stored seperate from the `revisions` list in a field called `draftRevision`. State users view and edit their draft in progress and this directly updates the `draftRevision`. CMS users, on the other hand, never see data from that field.
```

Comment:
So, this is a little confusing with the above statement.

At the Postgres Table level, draft revisions and submitted revisions live in the same table. A draft revision is created by createContract and unlockContract. The draft revision form data is updated by updateDraftContract(withRates), and submitContract creates a new updateInfo and adds it to the draft revision table, thus minting it as a submitted revision. 

In our domain models, the revision history only has submitted revisions. The (one max) draft revision if it exists is stored separately on the ContractType.